Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -738,6 +738,68 @@ def inlHom [Semiring R] [AddCommMonoid M] [Module R M] [Module Rᵐᵒᵖ M] : R
 
 end Mul
 
+section Inv
+
+variable {R : Type u} {M : Type v}
+
+instance inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M] : Inv (tsze R M) :=
```

Comment:
```suggestion
instance instInv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M] : Inv (tsze R M) :=
```
is the new convention. Only this name needs to change (because it is an instance)